Authentication/Authentication/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+
+namespace Authentication
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var path = @"./Data/Users.txt";
+
+            CheckDuplicateUsers(path);
+            
+            Menu(path);
+        }
+
+        static void Menu(string path)
+        {
+            var choice = "0";
+
+            while (!choice.Equals("Exit", StringComparison.OrdinalIgnoreCase))
+            {
+                choice = GetUserChoice();
+
+                if (choice.Equals("Login", StringComparison.OrdinalIgnoreCase))
+                {
+                    try
+                    {
+                        Login(path);
+                    }
+                    catch (FileNotFoundException exception)
+                    {
+                        throw new UsersNotFoundException(exception.FileName);
+                    }
+                    finally
+                    {
+                        Console.WriteLine();
+                    }
+                }
+                else if (choice.Equals("Register", StringComparison.OrdinalIgnoreCase))
+                {
+                    Register(path);
+                    Console.WriteLine();
```

Comment:
All parts end with `Console.WriteLine()`- consider moving outside of the if statement.